source/features/github-bugs.css

```diff
@@ -32,11 +13,6 @@
 	margin-left: 8px;
 }
 
-/* Avoid scroll bar on single-line `pre` in Markdown #4899 */
-.zeroclipboard-container clipboard-copy {
-	margin-bottom: 0 !important;
-}
```

Comment:
Test URL: https://github.com/VSCodium/vscodium#install-with-brew-mac

I'm not entirely sure about this one. The bug doesn't appear on Firefox, but seeing as scrollbars are also a native thing, this might not be the case on other browsers.

Reply:
This one happens more often when there's an emoji in a code block in a comment. It happens more often probably due to our own "limit code block height" feature

Reply:
> The bug doesn't appear on Firefox, but seeing as scrollbars are also a native thing, this might not be the case on other browsers.

I can still reproduce it. It does no harm so we can keep it for everyone.

> This one happens more often when there's an emoji in a code block in a comment.

I do find it is font-related. The issue appears when using the default Menlo but not JetBrains Mono.

> It happens more often probably due to our own "limit code block height" feature

It appears without Refined GitHub though.

Reply:
Ah I confused this with another bug. Disregard my comment.